sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -0,0 +1,119 @@
+# Azure Cognitive Services Text Analytics client library for .NET
+Azure Cognitive Services Text Analytics is a cloud service ithat provides advanced natural language processing over raw text. and includes six main functions: sentiment analysis, key phrase extraction, named entity recognition, linked entity recognition, recognition of personally identifiable information, and language detection.
+
+[Source code][textanalytics_client_src] | [Package (NuGet)][textanalytics_nuget_package] | [API reference documentation]() | [Product documentation][textanalytics_docs] | [Samples][textanalytics_samples]
```

Comment:
`API reference documentation` has no link

Reply:
Is `textanalytics_samples` link correct?

Reply:
Fixed the samples link - thanks!  API Reference docs won't be available until we release a package, I don't think.  I'll update this before then.